static/js/zamboni/devhub.js

```diff
@@ -124,6 +124,12 @@ $(document).ready(function() {
         $('.addon-upload-dependant').attr('disabled', !($(this).is(':checked')));
     });
 
+    $('#id_beta').bind('change', function() {
+        // TODO: Reset the form.
+        // I couldn't figure out how to do this easily. I was looking for a
+        // pre-existing function b/c I've seen this done elsewhere.
+    });
```

Comment:
@magopian I had trouble with this. I was hoping you could point me in the right direction? I know you had something when the user toggled between listed/unlisted that reset the form so they have to upload again. I was expecting to find some already bound method I could trigger but failed to find it.